presto-delta/src/main/java/com/facebook/presto/delta/DeltaSplit.java

```diff
@@ -118,12 +121,15 @@ public Map<String, String> getPartitionValues()
     @Override
     public NodeSelectionStrategy getNodeSelectionStrategy()
     {
-        return NO_PREFERENCE;
+        return nodeSelectionStrategy;
     }
 
     @Override
     public List<HostAddress> getPreferredNodes(NodeProvider nodeProvider)
     {
+        if (getNodeSelectionStrategy() == SOFT_AFFINITY) {
+            return nodeProvider.get(filePath, 2);
```

Comment:
2 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)).
Can you please add a comment on what 2 means.
An even better option is to refactor this to a constant var with a meaningful name.

Reply:
For the SOFT_AFFINITY node selection strategy scheduler would choose 2 workers (preferred, secondary preferred) for scheduling. This is similar to what we are doing in [HiveSplit](https://github.com/prestodb/presto/blob/master/presto-hive/src/main/java/com/facebook/presto/hive/HiveSplit.java#L163)

Reply:
Thanks for the explanation. !
would adding a const like this add some clarity ?
```
//  SOFT_AFFINITY node selection strategy scheduler would choose 2 workers (preferred, secondary preferred) for scheduling
private static final SOFT_AFFINITY_WORKER_COUNT = 2;
```

Reply:
I was thinking we can add a comment about it, and not update the variable. If there is just 1 worker then it would just have the preferred node option. Just to avoid any confusion with the variable name. Let me know what do you think?